apps/ledger-live-mobile/src/families/ethereum/EthereumCustomFees/utils.tsx

```diff
@@ -10,17 +13,39 @@ const defaultMaxFeePerGasRange = inferDynamicRange(TWENTY_GWEI); // 0 - 20 Gwei
 
 export const inferMaxPriorityFeeRange = (
   networkInfo: Transaction["networkInfo"],
+  transactionRaw?: TransactionRaw,
 ): Range => {
   if (!networkInfo?.maxPriorityFeePerGas || !networkInfo?.nextBaseFeePerGas)
     return defaultMaxPriorityFeeRange;
 
+  let minValue = getEnv("EIP1559_MINIMUM_FEES_GATE")
+    ? networkInfo.maxPriorityFeePerGas.min.times(
+        getEnv("EIP1559_PRIORITY_FEE_LOWER_GATE"),
+      )
+    : new BigNumber(0);
+
+  let maxValue = networkInfo.maxPriorityFeePerGas.max;
+
+  if (transactionRaw?.maxPriorityFeePerGas) {
+    const maxPriorityFeeGap: number = getEnv(
+      "EDIT_TX_EIP1559_MAXPRIORITYFEE_GAP_SPEEDUP_FACTOR",
+    );
+    const newMaxPriorityFeePerGas = new BigNumber(
+      transactionRaw.maxPriorityFeePerGas,
+    ).times(1 + maxPriorityFeeGap);
+
+    if (newMaxPriorityFeePerGas.isGreaterThan(new BigNumber(minValue))) {
+      minValue = newMaxPriorityFeePerGas;
+    }
+
+    if (newMaxPriorityFeePerGas.isGreaterThan(new BigNumber(maxValue))) {
```

Comment:
If I am not making mistake, the "newMaxPriorityFeePerGas" and maxValue/minValue is not in the same unit. We can't compare them like this.
newMaxPriorityFeePerGas is in wei while maxValue/minValue is in Gwei.
I've fixed that on LLD side. 
Can you check it on your side? Thanks

Reply:
Actually maxValue/minValue are in wei as well